### PR TITLE
Fix/wallet stuff

### PR DESCRIPTION
**Pull request by da-kami**
Sorry @Restioson but material3 does not seem to be a valid option for now. The trade screens look quite horrible and frankly it does not seem like this is ready yet. (the docs state that widgets that were not ported to material3 yet might behave / look unexpected).

I would stick to material2 for now, because that's proven, and deal with upgrading to material3 later.
I changed the `BalanceRow` to work with material2.

I was a bit confused about the padding / spacing of the buttons in relation to the row, so I simplified that.
For me it behaves as expected on Mac and iOS simulator:

---

Concerning the formatting and dart analyzer fixes: 
Unfortunately I merged this without the new `lint-flutter` action running on CI - for the future this should be detected by CI :)

### CODE REVIEW

**Review by scratchscratchscratchy — APPROVED** [review]
Looks good, but `BalanceRowButton` could be cleaned up better.

**Comment by da-kami** [comment]
Thanks @scratchscratchscratchy I was so hoping for a fast review!

File: mobile/pubspec.lock [hunk 1]
```diff
@@ -942,5 +942,5 @@ packages:
     source: hosted
     version: "2.0.3"
 sdks:
-  dart: ">=2.19.0 <4.0.0"
+  dart: ">=2.19.0 <3.0.0"
```
[the remark this answers is not in the document]
Comment (da-kami) [reply mobile/pubspec.lock:945]: Yeah, it's actually coming from not running `flutter pub get` - I'll see to create a separate commit.

[the remark this answers is not in the document]
Comment (da-kami) [reply mobile/pubspec.lock:945]: I wonder if I have an outdated dart version, let me check that

[the remark this answers is not in the document]
Comment (da-kami) [reply mobile/pubspec.lock:945]: Does not look like the version is outdated @Restioson any idea why this was not changed?
Potentially a package was removed but flutter pub get not run.

File: mobile/lib/features/wallet/balance_row.dart [hunk 2]
```diff
@@ -129,40 +146,34 @@ class BalanceRowButton extends StatelessWidget {
   final WalletType walletType;
```
[the remark this answers is not in the document]
Comment (da-kami) [reply mobile/lib/features/wallet/balance_row.dart:146]: Thanks, removed!

File: mobile/pubspec.lock (hunk 1 above)
[the remark this answers is not in the document]
Comment (Restioson) [reply mobile/pubspec.lock:945]: Not 100% sure unfortunately, I am a bit unfamiliar with how pub works

File: mobile/lib/features/wallet/balance_row.dart [hunk 3]
```diff
@@ -61,127 +59,134 @@ class _BalanceRowState extends State<BalanceRow>
...
                 ),
-                child: Row(
-                  children: [
+                receive
+              ]),
+            ),
+            PositionedTransition(
+              rect: RelativeRectTween(
+                      begin: RelativeRect.fill,
+                      end: RelativeRect.fromLTRB(buttonWidth * 2 + buttonSpacing * 2, 0, 0, 0))
```
Comment (Restioson) [inline mobile/lib/features/wallet/balance_row.dart:100]: This is a bit nicer, thanks :)

File: mobile/lib/features/wallet/balance_row.dart [hunk 4]
```diff
@@ -61,127 +59,134 @@ class _BalanceRowState extends State<BalanceRow>
...
-        },
-        tooltip: enabled ? "$action bitcoins on $type" : null,
-        icon: Icon(icon),
-        style: theme.iconButtonStyle,
-        padding: const EdgeInsets.all(horizontalPadding),
+    return SizedBox(
+      width: buttonSize,
+      child: ElevatedButton(
+        onPressed: !enabled
+            ? null
```
Comment (Restioson) [inline mobile/lib/features/wallet/balance_row.dart:171]: I find this ternary kinda weird; is this really preferred dart style?

**Pull request merged**